Source/WebCore/Modules/webxr/WebXRSession.cpp

```diff
@@ -110,6 +111,19 @@ const WebXRInputSourceArray& WebXRSession::inputSources() const
     return m_inputSources;
 }
 
+// https://www.w3.org/TR/webxr/#dom-xrsession-enabledfeatures
+const Vector<String> WebXRSession::enabledFeatures() const
+{
+    Vector<String> enabledFeatureArray;
+    for (const auto& feature : m_requestedFeatures) {
+        String sessionFeature = PlatformXR::sessionFeatureDescriptor(feature);
+        if (sessionFeature != ""_s)
+            enabledFeatureArray.append(sessionFeature);
```

Comment:
Can WTFMove the string here since it’s not used after the append call

Reply:
Changed, thank you!